reactor-kafka-samples/src/test/java/reactor/kafka/samples/SampleScenariosTest.java

```diff
@@ -96,8 +97,9 @@ public void kafkaSinkChain() throws Exception {
         sinkChain.runScenario();
         waitForMessages(expected, received1);
         List<Person> expected2 = new ArrayList<>();
-        for (Person p : expected)
+        for (Person p : expected) {
```

Comment:
nitpick: there seems to be unrelated formatting changes in this PR

Reply:
as far as I can see, no meaningful change but formatting in that file (and marking a field above as `final`), so I would lean towards removing the formatting just in this file

Reply:
nevermind that, there's a method call added to a method call chain below in `kafkaTransform`. I don't mind terribly, just a habit of avoiding large formatting changes in unmodified lines in other reactor projects.